projects/packages/forms/src/modules/form/view.js

```diff
@@ -0,0 +1,215 @@
+import { getContext, store, getConfig, withSyncEvent } from '@wordpress/interactivity';
```

Comment:
`withSyncEvent` is only available from WordPress 6.8 onwards, so if you need to maintain compatibility with previous versions, you should check that it exists.

Something like this:

```js
import { getContext, store, getConfig, withSyncEvent as originalWithSyncEvent } from '@wordpress/interactivity';

const withSyncEvent = originalWithSyncEvent || (cb => (...args) => cb(...args));
```

Reply:
Thanks for this suggestion!